onsstove/onsstove.py

```diff
@@ -250,7 +250,33 @@ def raster_to_dataframe(self, layer, method='sample'):
         elif method=='read':
             self.df[layer.name] = layer.layer[self.rows, self.cols]
 
-    
+    def calibrate_urban(self):
+
+        urban_modelled = 2
+        factor = 1
+        pop_tot = self.specs["Population_start_year"]
+        urban_current = self.specs["Urban_start"]
+
+        i = 0
+        while abs(urban_modelled - urban_current) > 0.01:
+
+            self.df["IsUrban"] = 0
+            self.df.loc[(self.df["Population"] > 300 * factor), "IsUrban"] = 1
```

Comment:
Make the parameters of 300 and 1500 resolution-independent, as we discussed we can use the area from the `self.cell_size` variable for this.